bundles/CoreBundle/src/DataCollector/PimcoreDataCollector.php

```diff
@@ -40,6 +46,21 @@ public function collect(Request $request, Response $response, ?\Throwable $excep
             'revision' => Version::getRevision(),
             'context' => $this->contextResolver->getPimcoreContext($request),
         ];
+
+        $element = $request->attributes->get('object') ?? $request->attributes->get(DynamicRouter::CONTENT_KEY);
```

Comment:
Thank you for this PR, and sorry for the long wait.

For documents look quite good, but doesn't work for objects, or at least for these dynamically generated (forgot the name) ones like
https://demo.pimcore.com/en/News/Montery-Car-Week-Spring-Edition~n765
These don't have an hardcoded name and don't work with the element but an id eg.
https://github.com/pimcore/demo/blob/4cd83c28782af1116a19bf2546c2d1c0b7489ad6/src/Website/LinkGenerator/NewsLinkGenerator.php#L95